Packages/com.vrchat.ClientSim/Editor/Windows/ClientSimSettingsWindow.cs

```diff
@@ -382,6 +383,7 @@ private void DrawPlayerControllerSettings()
                 _settings.invertMouseLook = EditorGUILayout.Toggle(_invertMouseLookGuiContent, _settings.invertMouseLook);
                 _settings.playerHeight = EditorGUILayout.FloatField(_playerHeightGuiContent, _settings.playerHeight);
                 _settings.playerHeight = Mathf.Clamp(_settings.playerHeight, 0.2f, 80f); // TODO make consts for these.
+                _settings.currentLanguage = EditorGUILayout.TextField(_currentLanguageGuiContent, _settings.currentLanguage);
```

Comment:
Could we make this a dropdown field instead of write-in?

Reply:
Good idea, done.
![language-dropdown](https://github.com/vrchat-community/ClientSim/assets/134413505/a3cb7254-07a1-4eee-a5e4-8950526678fc)